### PR TITLE
Add some changelog entries (based on PR descriptions)

### PR DESCRIPTION
**Pull request by vanchi-zendesk**
Also adds a test to catch if we miss it in future.

@grosser

### CODE REVIEW

**Comment by grosser** [comment]
nice, that got never released :D

File: test/test_changelog.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require_relative 'helper'
+
+describe Prop::VERSION do
+  it "should always add a changelog while bumping versions" do
+    changes = File.read("#{File.dirname(__FILE__)}/../Changelog.md")
+    assert changes.include?("## #{Prop::VERSION}"), "version #{Prop::VERSION}"\
```
Comment (grosser) [inline test/test_changelog.rb]:
```suggestion
    expected = "## #{Prop::VERSION}"
    assert changes.include?(expected), "#{expected} not found in Changelog.md"
```

**Comment by grosser** [comment]
if tests break because of bundler try `before_script: ruby -e "File.write('Gemfile.lock', File.read('Gemfile.lock').split('BUNDLED WITH').first)"`

**Comment by vanchi-zendesk** [comment]
Travis is failing to setup rvm some times for the jruby builds. Getting it passed by repeated attempts for now. Maybe we should remove testing for jruby.

**Pull request merged**